lib/exchange_api_web/controllers/user_controller.ex

```diff
@@ -0,0 +1,49 @@
+defmodule ExchangeApiWeb.UserController do
+  use ExchangeApiWeb, :controller
+
+  alias ExchangeApi.Accounts
+  alias ExchangeApi.Accounts.User
+  alias ExchangeApi.Guardian
+
+  action_fallback ExchangeApiWeb.FallbackController
+
+  def index(conn, _params) do
+    users = Accounts.list_users()
+    render(conn, "index.json", users: users)
+  end
+
+  def create(conn, %{"user" => user_params}) do
+    user_params = Map.put(user_params, "jwt", "")
+
+    with {:ok, %User{} = user} <- Accounts.create_user(user_params),
+         {:ok, token, _claims} <- Guardian.encode_and_sign(user) do
+      user_params = Map.put(user_params, "jwt", token)
+
+      Accounts.update_user(user, user_params)
```

Comment:
Our `create_user/1` could be responsible for creating a uuid and token and creating a user with these params, instead of having this logic here. Then for our response we can render the `user.jwt`